com.github.falkTX.Carla.json

```diff
@@ -0,0 +1,134 @@
+{
+  "app-id": "com.github.falkTX.Carla",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "carla",
+  "rename-desktop-file": "carla.desktop",
+  "rename-icon": "carla",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* For pipewire */
+    "--filesystem=xdg-run/pipewire-0",
+    "--system-talk-name=org.freedesktop.RealtimeKit1",
```

Comment:
I don't think this can work considering the PID namespace.  
Apps should use the [Realtime Portal](https://flatpak.github.io/xdg-desktop-portal/portal-docs.html#gdbus-org.freedesktop.portal.Realtime).  
If you added this for PipeWire (JACK) then I believe it's wrong and should be dropped.

Reply:
Some app crash if they don't have access to it - speaking from experience here. 

(can't speak for this one)

Reply:
@hfiguiere 
Thanks for pointing this out, I haven't hit this issue yet with the few JACK apps I packaged.

Reply:
maybe it got fixed since. But back when it was starting to work that was the case. And I don't really have the time to regress everything.

Reply:
I found this in most other jack apps, so put it here. Not sure if it is really still needed. Could try without. But it is hard to test for now because something isn't working anyway (Carla can't see other clients and devices).

Reply:
removed it for now because it doesn't crash Carla

Reply:
now that i got Carla fully working, I can confirm that this is not needed